andela/wger-prometheus
# [Feature #150397735] Clear exercise cache when a muscle is deleted

#### What does this PR do?
Reset the exercise cache when a muscle is deleted
#### Description of Task to be completed?
- When an administrator deletes a muscle from the muscle list it should not remain in an exercise that it has been added to
#### How should this be manually tested?
- Login as an administrator and create a new muscle under the exercises tab
- Create a new exercise and add the muscle to the exerecise
- Go back to the muscles view and delete the muscle
- The muscle should not appear on the muscle list in the exercise
#### Any background context you want to provide?
- Initially, when a muscle was deleted, it persisted in the exercise view of an exercise that it had been added to
#### What are the relevant pivotal tracker stories?
#150397735
